basic/basic.js

```diff
@@ -18,9 +24,34 @@ function entry() {
   }
 }
 
+function edit() {
+  if (field2.value === "") {
+    EditBtn.disabled = true;
+  } else {
+    EditBtn.disabled = false;
+    let edited = enters.filter((enter) => enter === field2.value);
+    console.log(edited);
+    while (arrayValue.length !== 0) {
```

Comment:
refactor:
teraz gdy masz tablicę przefiltrowanych elementów, to można przekształcić ją w tablicę elementów <li> (przy pomocy metody .map)
a następnie połączyć je i wstawić do znacznika ul